addons/account/wizard/setup_wizards.py

```diff
@@ -38,30 +39,40 @@ def _check_fiscalyear(self):
                     _('Incorrect fiscal year date: day is out of range for month. Month: %s; Day: %s') %
                     (wiz.fiscalyear_last_month, wiz.fiscalyear_last_day)
                 )
+
     @api.multi
     def action_save_onboarding_fiscal_year(self):
         self.env.user.company_id.set_onboarding_step_done('account_setup_fy_data_state')
 
-
 class SetupBarBankConfigWizard(models.TransientModel):
     _inherits = {'res.partner.bank': 'res_partner_bank_id'}
     _name = 'account.setup.bank.manual.config'
     _description = 'Bank setup manual config'
 
     res_partner_bank_id = fields.Many2one(comodel_name='res.partner.bank', ondelete='cascade', required=True)
-    create_or_link_option = fields.Selection(selection=[('new', 'Create new journal'), ('link', 'Link to an existing journal')], default='new')
-    new_journal_name = fields.Char(compute='compute_new_journal_related_data', inverse='set_linked_journal_id', required=True, help='Will be used to name the Journal related to this bank account')
-    linked_journal_id = fields.Many2one(string="Journal", comodel_name='account.journal', compute='compute_linked_journal_id', inverse='set_linked_journal_id')
-    new_journal_code = fields.Char(string="Code", required=True, default=lambda self: self.env['account.journal'].get_next_bank_cash_default_code('bank', self.env['res.company']._company_default_get('account.journal').id))
-
+    new_journal_name = fields.Char(default=lambda self:self.linked_journal_id.name, inverse='set_linked_journal_id', required=True, help='Will be used to name the Journal related to this bank account')
+    linked_journal_id = fields.Many2one(string="Journal", comodel_name='account.journal', inverse='set_linked_journal_id', compute="compute_linked_journal_id", default=lambda self:self.default_linked_journal_id())
```

Comment:
compute + default? Oo Can't we remove the default (to check)?

Reply:
The field is set to False without the default, not sure of the use of compute though

Reply:
I corrected it